packages/opds-web-client/src/components/BookCover.tsx

```diff
@@ -25,78 +26,42 @@ export default class BookCover extends React.Component<BookCoverProps, any> {
   }
 
   render() {
-    let bookCoverStyle = {
-      width: "150px",
-      height: "200px",
-      float: "left",
-      border: "1px solid #ccc"
-    };
-
     if (this.props.book.imageUrl && !this.state.error) {
       return (
         <img
           src={this.props.book.imageUrl}
           onError={this.handleError}
-          style={bookCoverStyle}
+          className="book-cover"
           alt=""
           />
       );
     }
 
     let { title, authors } = this.props.book;
 
-    let titleStyle = Object.assign({
-      display: "table-cell",
-      height: "120px",
-      width: "150px",
-      verticalAlign: "bottom",
-      color: "#444",
-      backgroundColor: "#eee",
-      fontWeight: "normal",
-      padding: "10px",
-    }, this.computeFontStyle(title, 40));
-
-    let authorStyle = Object.assign({
-      paddingTop: "10px",
-      color: "#fff",
-      padding: "10px"
-    }, this.computeFontStyle(authors.join(", "), 25));
+    let titleFontSize = this.computeFontSize(title, 40);
+    let authorFontSize = this.computeFontSize(authors.join(", "), 25);
 
     let hue = this.seededRandomHue(title);
     let bgColor = `hsla(${hue}, 40%, 60%, 1)`;
-    let cellStyle = {
-      float: "left",
-      width: "150px",
-      height: "200px",
-      backgroundColor: bgColor,
-      border: "1px solid #888",
-      fontWeight: "normal",
-      fontFamily: "Georgia, serif",
-      position: "relative",
-      textAlign: "left"
-    };
 
     return (
-      <div className="autoBookCover" style={cellStyle}>
-        <div style={titleStyle}>{ title }</div>
-        <div style={authorStyle}>{ authors.join(", ") }</div>
+      <div className="auto-book-cover" style={{ backgroundColor: bgColor }}>
```

Comment:
yeah, it picks a random color for the cover and tries to pick a font size that will make the title and author fit.